bind_paths_and_mounts.rst

```diff
@@ -249,32 +261,58 @@ Using ``--no-home`` and ``--containall`` flags
 ``--no-home``
 -------------
 
-When shelling into your container image, {Project} allows you to
-mount your current working directory (``CWD``) without mounting your
-host ``$HOME`` directory with the ``--no-home`` flag.
+When starting a container, {Project} allows you to mount your current
+working directory (``CWD``) without mounting your host ``$HOME`` directory by
+using the ``--no-home`` flag. This is equivalent to ``--no-mount home``:
 
 .. code::
 
    $ {command} shell --no-home my_container.sif
 
+   -or-
+
+   $ {command} shell --no-mount home my_container.sif
+
+Disabling the mount of ``$HOME`` may be useful if your container image has files
+at ``$HOME``, which would otherwise be hidden by the bind mount from the host.
+
 .. note::
 
-   Beware that if it is the case that your ``CWD`` is your ``$HOME``
-   directory, it will still mount your ``$HOME`` directory.
+  If your current working directory is under ``$HOME``, and you do not want to
+  mount it, you will need to disable both ``cwd`` and ``home`` mounts:
+
+  .. code::
+
+    $ {command} shell --no-mount home,cwd my_container.sif
+
+``--contain`` / ``--containall``
+--------------------------------
 
-``--containall``
-----------------
+When using the ``--contain`` / ``--containall`` (or ``-c`` / ``-C`` for short)
+flags, ``$HOME`` from the host is not mounted, and an in-memory temporary
+directory is created at the ``$HOME`` point inside the container instead. An
+in-memory temporary directory is also used for ``/tmp`` and ``/var/tmp`` inside
+the container.
 
-   Using the ``--containall`` (or ``-C`` for short) flag, ``$HOME`` is
-   not mounted and a dummy bind mount is created at the ``$HOME`` point.
-   You cannot use ``-B``` (or ``--bind``) to bind your ``$HOME``
-   directory because it creates an empty mount. So if you have files
-   located in the image at ``/home/user``, the ``--containall`` flag
-   will hide them all.
+If the container image includes files within ``$HOME``, the mounted temporary
+directory will hide them unless you also specify ``--no-home`` or ``--no-mount
+home``:
 
 .. code::
 
    $ {command} shell --containall my_container.sif
+   Apptainer> ls -lah $HOME
```

Comment:
Use `{Project}` instead of `Apptainer` here, and below.

Reply:
thanks! updated.